flutter/lib/src/navigation/sentry_navigator_observer.dart

```diff
@@ -365,12 +355,11 @@ class SentryNavigatorObserver extends RouteObserver<PageRoute<dynamic>> {
         return;
       }
 
-      if (isAppStart && endTimestamp != null) {
-        await _timeToDisplayTracker?.trackAppStartTTD(transaction,
```

Comment:
can we also remove `trackAppStartTTD` from the `TimeToDisplayTracker` class and from `TimeToInitialDisplayTracker`?

might also make sense to make a comment in `TimeToDisplayTracker` that app starts are managed solely by the app start integration